src/modules/85-iac/components/IacApp/index.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+/* eslint-disable import/no-unresolved */
+import React, { lazy } from 'react'
+import { customComponents, customFunctions, customHooks } from '@iac/utils/IacChildAppUtils'
+import ChildAppMounter from 'microfrontends/ChildAppMounter'
+import type { IacmCustomMicroFrontendProps } from '@iac/interfaces/IacCustomMicroFrontendProps.types'
+
+const RemoteIacmApp = lazy(() => import('iacm/MicroFrontendApp'))
+
+export const IacmApp = (): React.ReactElement => (
+  <ChildAppMounter<IacmCustomMicroFrontendProps>
```

Comment:
Try to keep `IACM` capitalised everywhere by default
```suggestion
const RemoteIACMApp = lazy(() => import('iacm/MicroFrontendApp'))

export const IACMApp = (): React.ReactElement => (
  <ChildAppMounter<IACMCustomMicroFrontendProps>
```